include/mbgl/style/filter.hpp

```diff
@@ -11,14 +11,16 @@
 
 namespace mbgl {
 namespace style {
-    
+
 class Filter {
 public:
     optional<std::shared_ptr<const expression::Expression>> expression;
-    
+    optional<mbgl::Value> filter;
```

Comment:
Let's rename this slightly more descriptively -- maybe `legacyFilterValue` or something?

Reply:
This should probably be private, too